docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -95,147 +95,39 @@ but timeouts seem to be measured in seconds.
     and code the functionality in Deno script,
     which is similar to JavaScript and Typescript.
     See
-    [Yaml synopsis for deno-runtime container](#yaml-synopsis-for-deno-runtime-container).
+    [Synopsis for deno-runtime container](#deno-runtime)
     * **python** -- Use a `python-runtime` function
     and code the functionality in Python 3.
     See
-    [Yaml synopsis for python-runtime runner](#yaml-synopsis-for-python-runtime-runner)
+    [Synopsis for python-runtime runner](#python-runtime)
     * **container** -- Use the runner defined
       for the `container-runtime` container.
       This is a standard Kubernetes container
       for which you define the image, runner, runtime parameters, etc.
       and code the functionality to match the container you define.
       See
-      [Yaml synopsis for container-runtime container](#yaml-synopsis-for-container-runtime).
-  * **retries** (optional) - specifies the number of times,
+      [Synopsis for container-runtime container](#synopsis-for-container-runtime).
+  * **retries** (optional) -- specifies the number of times
     a job executing the `KeptnTaskDefinition`
     should be restarted if an attempt is unsuccessful.
-  * **timeout** (optional)* -- specifies the maximum time, in seconds,
+  * **timeout** (optional) -- specifies the maximum time, in seconds,
```

Comment:
this does not necessarily have to be in seconds, as this field accepts any valid duration string (e.g. could also be `5m`)

Reply:
fixed